ArduPlane/Parameters.h

```diff
@@ -344,7 +344,8 @@ class Parameters {
 
         k_param_DataFlash = 253, // Logging Group
 
-        // 254,255: reserved
+		k_param_swarm_failsafe_enable,
+		// 255: reserved
```

Comment:
as @OXINARF said, please use g2 and keep these reserved.

Reply:
when I used astyle --options=Tools/CodeStyle/astylerc <filename>.It shows many formatting changes.I don't know whether is ok for this commit.If something wrong with this.I really feel apologetic about that.

Reply:
when I used astyle --options=Tools/CodeStyle/astylerc <filename>.It shows many formatting changes.I don't know whether is ok for this commit.If something wrong with this.I really feel apologetic about that.